ext/mysql2/statement.h

```diff
@@ -13,6 +13,6 @@ void init_mysql2_statement(void);
 void decr_mysql2_stmt(mysql_stmt_wrapper *stmt_wrapper);
 
 VALUE rb_mysql_stmt_new(VALUE rb_client, VALUE sql);
-void rb_raise_mysql2_stmt_error(mysql_stmt_wrapper *stmt_wrapper);
+void rb_raise_mysql2_stmt_error(mysql_stmt_wrapper *stmt_wrapper) __attribute__ ((noreturn));
```

Comment:
aren't unknown attributes ignored by compilers that don't support them? what did the build do on appveyor?